.github/workflows/dart_frog_prod_server.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
   build:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
     with:
+      runs_on: macos-latest
```

Comment:
Did u figure out why the behavior was different?

Reply:
Yeah, I was going to bring it up at standup but for now I wanted to unblock this so we can continue with the upgrades